### DISCUSSION TITLE
Разрушаемые объекты

### DISCUSSION THREAD

**Discussion by me2beats** [godot-engine-ru/shooter-3d#20] [Ideas]
Коробки, ящики можно разрушать
Двери выбивать (не все, только те которые можно по миссии)

**Comment by Pigzinzspace** [comment]
а чеб не написать это в геймплэй или все предложения по игре

**Reply by me2beats** [reply]
> а чеб не написать это в геймплэй или все предложения по игре

Думаю важно определиться с этим, что будет разрушаемым, а что не будет

**Reply by Pigzinzspace** [reply]
я добавил в геймплэй строчку, просто не надо создавать отдельные темы, пиши предложения в комменты 
сюда
https://github.com/godot-engine-ru/shooter-3d/discussions/12 
или сюда 
https://github.com/godot-engine-ru/shooter-3d/discussions/18
